lib/workers/repository/process/index.ts

```diff
@@ -103,6 +104,11 @@ export async function extractDependencies(
     logger.debug('No baseBranches');
     const packageFiles = await extract(config);
     addSplit('extract');
+    if (GlobalConfig.get('dryRun') === 'extract') {
+      res.packageFiles = packageFiles;
+      logger.debug({ packageFiles }, 'Extracted dependencies');
```

Comment:
```suggestion
      logger.info({ packageFiles }, 'Extracted dependencies');
```

I think we may as well INFO this for this special case